src/mlpack/methods/emst/union_find.hpp

```diff
@@ -101,4 +101,4 @@ class UnionFind
 } // namespace emst
 } // namespace mlpack
 
-#endif // MLPACK_METHODS_EMST_UNION_FIND_HPP
+#endif // MLPACK_METHODS_EMST_UNION_FIND_HPP
```

Comment:
No need to change the line ending on this file---it's probably good to leave the line endings there.  Can you revert this? :)